### PR TITLE
CMDCT-3660: Refactor HTML in JSON Templates

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Refactoring the `mcpar.json` and `mlr.json` templates to make the HTML cleaner.

### Related ticket(s)

CMDCT-3660

---
### How to test

Verbiage should look exactly the same, no changes in functionality :)

### Notes

Updates `mcpar.json` and `mlr.json` templates.

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/app-api/forms/mcpar.json [hunk 1]
```diff
@@ -2689,8 +2752,8 @@
               "spreadsheet": "D2_Plan_Measures",
               "info": [
                 {
-                  "type": "html",
-                  "content": "<p>Report on individual measures in each of the following eight domains: (1) Primary care access and preventive care, (2) Maternal and perinatal health, (3) Care of acute and chronic conditions, (4) Behavioral health care, (5) Dental and oral health services, (6) Health plan enrollee experience of care, (7) Long-term services and supports, and (8) Other. For composite measures, be sure to include each individual sub-measure component.</p>"
+                  "type": "p",
```
Comment (rocio-desantiago) [inline services/app-api/forms/mcpar.json:2755]: Does this one not need the `children` prop like the ones above?

File: services/app-api/forms/mcpar.json [hunk 2]
```diff
@@ -3420,7 +3488,7 @@
               "spreadsheet": "D4_Plan_ILOS",
               "info": [
                 {
-                  "type": "span",
+                  "type": "p",
```
Comment (rocio-desantiago) [inline services/app-api/forms/mcpar.json:3491]: Same here? I may not be understand the structure breakdown for when the `children` prop needs to be passed.

**Review by rocio-desantiago — APPROVED** [review]
Just a question behind the children props where some p tags have them and others don't.

File: services/app-api/forms/mcpar.json (hunk 1 above)
Comment (gmrabian) [reply services/app-api/forms/mcpar.json:2755]: I believe the `children` prop is just for when there are multiple elements included in the paragraph, like when we have an embedded link. If it's just a single chunk of text with no additional elements, it doesn't need the `children`

Comment (rocio-desantiago) [reply services/app-api/forms/mcpar.json:2755]: Ah gotcha.

**Comment by karla-vm** [comment]
> Just a question behind the children props where some p tags have them and others don't.

good question! I think what @gmrabian mentioned is exactly it 💯

**Pull request merged**